todoman/model.py

```diff
@@ -916,15 +916,15 @@ def colour_for_path(path):
         try:
             with open(os.path.join(path, 'color')) as f:
                 return f.read().strip()
-        except (OSError, IOError):
+        except (IOError):
```

Comment:
Can you explain this change? It's not obvious to me why `OSError` shouldn't be here.

Also, parens are redudant.

```suggestion
        except IOError:
```

Reply:
This changes originates in the flake8 report generated by running `tox -e flake8`, which marks this as redundant and suggests using `except ():`, which of course is wrong. After some digging around, as mentioned in the commit, I came across an issue in the flake8 repo explaining this warning: in python2, IOError and OSError are different, but in python3, they actually resolve to the same object:

> Changed in version 3.3: EnvironmentError, IOError, WindowsError, socket.error, select.error and mmap.error have been merged into OSError, and the constructor may return a subclass.

See the discussion in [the issue](https://github.com/PyCQA/flake8-bugbear/issues/110) for details.

Since we only support python3, it makes sense to only use OSError. And of course, because brain, I wrote IOError. What we want is `except OSError:`, I'll fix this.

Reply:
See 530dc3652445ffb88ee28ed90a4366a411e2ffa2 for the correct change ^^ And applying this PR should also fix the flake8 warning you get in #375.

Reply:
Ah, thanks for the background on this, good to know!